test/parallel/test-tls-psk-circuit.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const tls = require('tls');
+
+const CIPHERS = 'PSK+HIGH';
```

Comment:
Is this the only cipher suite, simply `PSK`? If so, we should document its format in the API docs.

Reply:
There are multiple available ciphers that can be obtained via `openssl ciphers -V 'PSK'`. Though not all of those are supported, those with RSA, DHE, ECDHE wouldn't work.